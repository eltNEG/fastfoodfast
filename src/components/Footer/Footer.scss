@import '../../styles/styles';

.footer {
    height: 5rem;
    color: $white;
    background-color: $dark-blue;
    display: flex;
    align-items: center;
    justify-content: center;
}