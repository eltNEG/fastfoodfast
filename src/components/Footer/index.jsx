import React from 'react';
import './Footer.scss';

export default () => <div className="footer">Â©2018 Fast Food Fast - Andela Bootcamp challenge</div>;
