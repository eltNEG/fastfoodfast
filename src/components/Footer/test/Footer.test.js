import React from 'react';
import { shallow } from 'enzyme';
import Footer from '..';

describe('Footer.jsx', () => {
  it('should match Footer.jsx snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
