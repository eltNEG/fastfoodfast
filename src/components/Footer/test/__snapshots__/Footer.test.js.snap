// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.jsx should match Footer.jsx snapshot 1`] = `
<div
  className="footer"
>
  Â©2018 Fast Food Fast - Andela Bootcamp challenge
</div>
`;
